addOns/automation/CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Fixed
 - Templates generated with `-autogenmin` or `-autogenmax` were invalid in some cases.
 - Allow to choose one thread for the `activeScan` job through the GUI.
+- Active Scan jobs will now properly use policyDefinition components.
```

Comment:
`… use the default policy.`

Reply:
The `policyDefinition` was working fine, not sure why mention it. Also whether it's Default Policy or not depends on what the default is (though we do call it that way in the templates…).